Samples/Tutorial Samples/YOLOv4ObjectDetection/YOLOv4ObjectDetection/MainPage.xaml.cs

```diff
@@ -331,46 +175,43 @@ private async Task DrawBoxes(float[] results, VideoFrame frame)
                 textBlock.Foreground = foregroundColorBrush;
                 textBlock.FontSize = 18;
 
-                textBlock.Text = final_detetions[i].label;
+                textBlock.Text = detections[i].label;
                 // Hide
                 textBlock.Visibility = Visibility.Collapsed;
                 border.Background = backgroundColorBrush;
                 border.Child = textBlock;
 
-                Canvas.SetLeft(border, final_detetions[i].bbox[1] * 416 + 2);
-                Canvas.SetTop(border, final_detetions[i].bbox[0] * 416 + 2);
+                Canvas.SetLeft(border, detections[i].bbox[1] * 416 + 2);
+                Canvas.SetTop(border, detections[i].bbox[0] * 416 + 2);
                 textBlock.Visibility = Visibility.Visible;
                 // Add to canvas
                 this.OverlayCanvas.Children.Add(border);
             }
         }
 
-        private async Task<List<float>> EvaluateFrame(VideoFrame frame)
+        private void button_go_Click(object sender, RoutedEventArgs e)
         {
-            _binding.Clear();
-            _binding.Bind("input_1:0", frame);
-            var results = await _session.EvaluateAsync(_binding, "");
-
-            TensorFloat result = results.Outputs["Identity:0"] as TensorFloat;
-            var shape = result.Shape;
-            var data = result.GetAsVectorView();
-            
-            return data.ToList<float>();
+            if (_timer == null)
+            {
+                // now start processing frames, no need to do more than 30 per second!
+                _timer = new DispatcherTimer()
+                {
+                    Interval = TimeSpan.FromMilliseconds(30)
+                };
+                _timer.Tick += OnTimerTick;
```

Comment:
are you missing a call to _timer.Start()? I built this locally and the tick event never got triggered.

Reply:
woops, yeah, currently I'm having trouble with the checked in Yolo model, see https://github.com/microsoft/Windows-Machine-Learning/issues/404.  If you have a model that works I'd like to try it...